src/common/options.cc

```diff
@@ -5414,8 +5414,17 @@ std::vector<Option> get_global_options() {
 
     Option("crimson_osd_scheduler_concurrency", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
     .set_default(0)
-    .set_description("The maximum number concurrent IO operations, 0 for unlimited")
+    .set_description("The maximum number concurrent IO operations, 0 for unlimited"),
 
+    // ----------------------------
+    // blk specific options
+    Option("block_device_type", Option::TYPE_STR, Option::LEVEL_ADVANCED)
+    .set_description("Explicitly set the device type to select the driver if it's needed")
```

Comment:
might want to use `set_enum_allowed()` here.

Reply:
good suggestion

Reply:
Done